pkg/tls/certificatemanagement/interface.go

```diff
@@ -23,12 +23,19 @@ import (
 const (
 	TenantCASecretName                = "tigera-ca-private-tenant"
 	CASecretName                      = "tigera-ca-private"
-	TrustedCertConfigMapName          = "tigera-ca-bundle"
 	TrustedCertConfigMapKeyName       = "tigera-ca-bundle.crt"
 	TrustedCertVolumeMountPath        = "/etc/pki/tls/"
 	TrustedCertVolumeMountPathWindows = "c:/etc/pki/tls/"
 	TrustedCertBundleMountPath        = "/etc/pki/tls/certs/tigera-ca-bundle.crt"
 	TrustedCertBundleMountPathWindows = "c:/etc/pki/tls/certs/tigera-ca-bundle.crt"
+
+	// TrustedCertConfigMapName is the name of the trusted certificate bundle ConfigMap. This value is used
+	// for all single-tenant trusted bundles, as well as multi-tenant trusted bundles that do not include public CAs.
+	TrustedCertConfigMapName = "tigera-ca-bundle"
+
+	// TrustedCertConfigMapNamePublic is the name of the trusted certificate bundle ConfigMap that includes public CAs, used
+	// only in multi-tenant environments as a single namespace requires both a trusted bundle with public CAs as well as one without.
+	TrustedCertConfigMapNamePublic = "tigera-ca-bundle-public"
```

Comment:
Probably want to rename this to `tigera-ca-bundle-with-system-roots`

Reply:
I see that for a single tenant, we should use the same config map `tigera-ca-bundle`, but with the following fields
- `tigera-ca-bundle.crt` holds certificates generated by tigera ca
- `ca-bundle.crt`holds public certificates

Why can't we use the same mechanism ?

Reply:
We don't want to mount the public certificates into ever pod, and this approach just seemed the most in line with how we currently manage these for enterprise. I think it's a bit easier to grasp conceptually.